code/modules/admin/admin_verbs.dm

```diff
@@ -296,20 +296,11 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 		/client/proc/readmin
 		)
 
-/client/proc/hide_most_verbs()//Allows you to keep some functionality while hiding some verbs
```

Comment:
why did you remove this

Reply:
Apparently because Im over tired... Ill add it back

Reply:
tbh looks pretty useless

Reply:
Oh now I see, I removed too much, One extra line got caught in the delete of the "Hide most"